app/models/manageiq/providers/openstack/cloud_manager.rb

```diff
@@ -181,4 +181,19 @@ def vm_delete_evm_snapshot(vm, image_id)
   def self.event_monitor_class
     ManageIQ::Providers::Openstack::CloudManager::EventCatcher
   end
+
+  #
+  # Statistics
+  #
+
+  # gigabytes
+  def cinder_disk_usage
```

Comment:
I see, I'll remove the comments. I was confused by the UI displaying different units. It was because of the test data I generated, gigabytes for cinder volumes, and bytes for swift objects.